shipgen/MuonBackGenerator.cxx

```diff
@@ -183,11 +184,13 @@ Bool_t MuonBackGenerator::ReadEvent(FairPrimaryGenerator* cpg)
          px = track->GetPx();
          py = track->GetPy();
          pz = track->GetPz();
-         if (fPaintBeam > 0) {
+         if (fPhiRandomize) {
+             gRandom->SetSeed(12345);  // TO BE FIXED: this should not be hardcoded
```

Comment:
Remove this line for now, as it will also affect the ability to reproduce the muon background.
If we can't set an independent seed, it's better to not set it.